hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerPrepare.java

```diff
@@ -338,11 +339,9 @@ public void testCancelPrepare() throws Exception {
   }
 
   private boolean logFilesPresentInRatisPeer(OzoneManager om) {
-    String ratisDir = om.getOmRatisServer().getServer().getProperties()
-        .get("raft.server.storage.dir");
-    String groupIdDirName =
-        om.getOmRatisServer().getServer().getGroupIds().iterator()
-            .next().getUuid().toString();
+    final RaftServer.Division server = om.getOmRatisServer().getServerDivision();
+    final String ratisDir = server.getRaftServer().getProperties().get("raft.server.storage.dir");
+    final String groupIdDirName = server.getGroup().getGroupId().toString();
```

Comment:
`TestOzoneManagerPrepare` is failing due to mismatch in the directory name.

```suggestion
    final String groupIdDirName = server.getGroup().getGroupId().getUuid().toString();
```

Reply:
That is a good catch!  Thanks for the tip.